File: src/client_gen/java/DebugInterface.java
import java.io.IOException;
import java.io.OutputStream;

public class DebugInterface {
    private OutputStream stream;

    public DebugInterface(OutputStream stream) {
        this.stream = stream;
    }

    public void send(model.DebugCommand command) {
        try {
            new model.ClientMessage.DebugMessage(command).writeTo(stream);
            stream.flush();
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/client_gen/kotlin/DebugInterface.kt
import java.io.IOException
import java.io.OutputStream

class DebugInterface(private val stream: OutputStream) {
    fun send(command: model.DebugCommand) {
        try {
            model.ClientMessage.DebugMessage(command).writeTo(stream)
            stream.flush()
        } catch (e: IOException) {
            throw RuntimeException(e)
        }
    }
}
